lib/plugins/health.js

```diff
@@ -24,6 +24,7 @@ function inject (bot) {
         bot.isAlive = false
         bot.emit('death')
       }
+      if (!bot.respawn) return
```

Comment:
What are the side effects of this?

Reply:
None that i know, please tell me if there are some

Reply:
> What are the side effects of this?
> None that i know, please tell me if there are some

it appears that there's no way to manually trigger the bot respawning aside from calling `bot._client.write('client_command', { payload: 0 })`

perhaps a `bot.respawn()` command would be useful?